doc/api/crypto.md

```diff
@@ -2422,6 +2422,25 @@ const ciphers = crypto.getCiphers();
 console.log(ciphers); // ['aes-128-cbc', 'aes-128-ccm', ...]
 ```
 
+### `crypto.getCipherInfo(nameOrNid)`
+<!-- YAML
+added: REPLACEME
+-->
+
+* `nameOrNid`: {string|number} The name or nid of the cipher to query.
+* Returns: {Object}
+  * `name` {string} The name of the cipher
+  * `nid` {number} The nid of the cipher
+  * `blockSize` {number} The block size of the cipher.
+  * `ivLength` {number} The expected or default initialization vector length or
+    zero if the cipher does not use an initialization vector.
+  * `keyLength` {number} The expected or default key length.
```

Comment:
The documentation should say whether these values are measured in bits or bytes.